src/controllers/auth.js

```diff
@@ -64,7 +65,7 @@ async function confirmed (req, res, next, client = FxAOAuthClient) {
 
   // Check if user is signing up or signing in,
   // then add new users to db and send email.
-  if (!existingUser || existingUser.fxa_refresh_token === null) {
+  if (!existingUser) {
     // req.session.newUser determines whether or not we show "fxa_new_user_bar" in template
     req.session.newUser = true
     const signupLanguage = req.headers['accept-language']
```

Comment:
I know this is beyond the scope of this PR, but for future we might consider using `req.appLocale` instead of `req.headers['accept-language']`.  `req.appLocale` is set via middleware in `app.js` and is supposed to intelligently negotiate a language that meets the request and that Monitor supports.  In contrast, the pure header as shown here might store multiple locales or even `*` or `null`.

Just wanted to share in case you or I run into this in the future!

Reply:
Good to know, let me change this